docs/GETTING_STARTED.md

```diff
@@ -67,11 +65,9 @@ Create namespace `local-ozone`, and deploy Ozone S3:
 
     $ sh ./setup-ozone-s3.sh
 
-This can take a few minutes. Ensure all the pods are running before moving on to the next step.
-
-    $ kubectl -n local-ozone get pods
+This can take a few minutes. Ensure all the pods are running for 1 minute before moving on to the next step.
```

Comment:
For a couple of times, after all pods running, tried running `init-s3-bucket.sh` immediately and always saw 500 error. Waiting for 1 min or rerun `init-s3-bucket.sh` will work.